openedx/core/lib/tests/test_time_zone_utils.py

```diff
@@ -0,0 +1,95 @@
+"""Tests covering time zone utilities."""
+from django.test import TestCase
```

Comment:
Question/Nit:  (cc: @nasthagiri) if our /lib code is supposed to be django-free, should this use unittest.TestCase instead?